app/app.py

```diff
@@ -0,0 +1,190 @@
+            dark=True,
+        ),
+    ),
+)
+
+# ------------------ Uploader ------------------ #
+# Configure the upload folder
+TEMP_DIR = tempfile.mkdtemp()
+du.configure_upload(app, TEMP_DIR)
+
+uploader = html.Div(
+    [
+        dbc.Row(
+            dbc.Col(
+                du.Upload(
+                    id="dash-uploader",
+                    text="Import Data",
+                    text_completed="Uploaded: ",
+                    filetypes=["pkl"],
+                    upload_id=uuid.uuid1(),  # Unique session id
+                    cancel_button=True,
+                    max_files=1,
+                ),
+            )
+        ),
+        dbc.Row(
+            dbc.Col(
+                html.Div(children="No file uploaded", id="dash-uploader-output", className="p-4"),
+                className="d-flex justify-content-center",
+            )
+        ),
+        dcc.Store(id="file-store"),  # Store to keep the file contents
+    ],
+    className="p-5 ml-5 mr-5",
+)
+
+# ------------------ Tabs ------------------ #
+# gm tab content
+gm_content = dbc.Row(
+    dbc.Col(
+        dbc.Card(
+            dbc.CardBody([html.Div(id="file-content-gm")]),
+        )
+    )
+)
+# mg tab content
+mg_content = dbc.Row(
+    dbc.Col(
+        dbc.Card(
+            dbc.CardBody([html.Div(id="file-content-mg")]),
+        )
+    ),
+)
+# tabs
+tabs = dbc.Row(
+    dbc.Col(
+        dbc.Tabs(
+            [
+                dbc.Tab(
+                    gm_content,
+                    label="Genomics -> Metabolomics",
+                    activeTabClassName="fw-bold",
+                    disabled=True,
+                    id="gm-tab",
+                    className="disabled-tab",
+                ),
+                dbc.Tab(
+                    mg_content,
+                    label="Metabolomics -> Genomics",
+                    activeTabClassName="fw-bold",
+                    disabled=True,
+                    id="mg-tab",
+                    className="disabled-tab",
+                ),
+            ],
+        ),
+    ),
+    className="p-5",
+)
+
+app.layout = dbc.Container([navbar, uploader, tabs], fluid=True, className="p-0")
+
+clientside_callback(
+    """
+    (switchOn) => {
+       document.documentElement.setAttribute('data-bs-theme', switchOn ? 'light' : 'dark');
+       return window.dash_clientside.no_update
+    }
+    """,
+    Output("color-mode-switch", "id"),
+    Input("color-mode-switch", "value"),
+)
+
+
+@du.callback(
+    id="dash-uploader",
+    output=[Output("dash-uploader-output", "children"), Output("file-store", "data")],
+)
+def upload_data(status: du.UploadStatus):  # noqa: D103
+    if status.is_completed:
+        latest_file = status.latest_file
+        with open(status.latest_file, "rb") as f:
+            pickle.load(f)
+        return (
+            f"Successfully uploaded file `{os.path.basename(latest_file)}` of size {round(status.uploaded_size_mb, 2)} MB.",
```

Comment:
```suggestion
            f"Successfully uploaded: {os.path.basename(latest_file)} [{round(status.uploaded_size_mb, 2)} MB]",
```
Trying to make the message simpler